packages/flame/lib/src/game/mixins/keyboard.dart

```diff
@@ -70,7 +70,7 @@ mixin KeyboardEvents on Game {
     Set<LogicalKeyboardKey> keysPressed,
   ) {
     assert(
-      this is HasKeyboardHandlerComponents,
+      this is! HasKeyboardHandlerComponents,
```

Comment:
was this wrong before? the message seem to imply that this should be just `is` right?

Reply:
ah I see I misread the `assert` as an `if`. how did we never caught this??

Reply:
no idea as well, this confused me a little bit too, I added a test for this now